src/Symfony/Bridge/Twig/Extension/FormExtension.php

```diff
@@ -119,7 +123,11 @@ public function __get($name)
             if (is_array($this->renderer)) {
                 $renderer = $this->renderer[0]->get($this->renderer[1]);
                 if (isset($this->renderer[2])) {
-                    $renderer->setEnvironment($this->renderer[2]);
+                    if ($renderer instanceof TwigRendererInterface) {
+                        $renderer->setEnvironment($this->renderer[2]);
+                    } elseif ($renderer instanceof FormRendererInterface && $renderer->getEngine() instanceof TwigRendererEngineInterface) {
```

Comment:
this case should never happen

Reply:
It can happen if you call `->initRuntime` before accessing `->renderer` on 3.4?

```php
$twig = $this->get('twig');

$twig
    ->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
    ->initRuntime($twig);

$twig
    ->getExtension('Symfony\Bridge\Twig\Extension\FormExtension')
    ->renderer;
```
Results in:
```
Attempted to call an undefined method named "setEnvironment" of class "Symfony\Component\Form\FormRenderer".
```

This works fine on 3.3.